Dataset_generation.m

```diff
@@ -754,27 +781,29 @@
 
 x = "Oscillatory Transient + Interruptions";                            
 t = [0: ts :0.2-ts];                   % 640 sample points per disturbance
-f = 50;
 
-for alpha=0.1:0.0773480663:0.9                         % Runs 10 times
+
+for alpha=0.1:0.2051282051:0.9                         % Runs 4 times
     for F_t=300:1160.493827:5000                    % Runs 5 times
-        for t3=0.04:0.01:0.08                   % Runs 5 times
-            for tau=0.008:0.007901234568:0.040          % Runs 5 times
-                t4=t3+0.02;          % 1 cycle
-                y= (1-alpha*((heaviside(t-t1)-heaviside(t-t2)))) * (alpha*(heaviside(t-t3)-heaviside(t-t4)).*exp(t3-t/tau).*sin(2*pi*F_t*t));
-                z= vertcat(z,y);
-                cl=vertcat(cl,x);
- %{               
-                t4=t3+0.03;          % 1.5 cycles
-                y= (1-alpha*((heaviside(t-t1)-heaviside(t-t2)))) + sin(2*pi*f*t)+ alpha*(heaviside(t-t3)-heaviside(t-t4)).*exp(t3-t/tau).*sin(2*pi*F_t*t);
-                z= vertcat(z,y);
-                cl=vertcat(cl,x);
-                
-                t4=t3+0.04;          %2 cycles
-                y= (1-alpha*((heaviside(t-t1)-heaviside(t-t2)))) + sin(2*pi*f*t)+ alpha*(heaviside(t-t3)-heaviside(t-t4)).*exp(t3-t/tau).*sin(2*pi*F_t*t);
-                z= vertcat(z,y);
-                cl=vertcat(cl,x);
-                %}
+        for f=49.9:0.1:50                        % Runs 2 times
+            for t3=0.04:0.01:0.08                   % Runs 5 times
+                for tau=0.008:0.007901234568:0.040          % Runs 5 times

```

Comment:
Can you check the step size.

Reply:
for t3 = 0.04, 0.05, 0.06, 0.07, 0.08
for tau ill chage